frontend/javascripts/oxalis/model/sagas/isosurface_saga.ts

```diff
@@ -658,7 +658,11 @@ function* loadPrecomputedMeshForSegmentId(
         getBaseSegmentationName(segmentationLayer),
         meshFileName,
         id,
-        mappingName,
+        // The back-end should only receive a non-null mapping name,
+        // if it should perform extra (reverse) look ups to compute a mesh
+        // with a specific mapping from a mesh file that was computed
+        // without a mapping.
+        meshFile.mappingName != null ? null : mappingName,
```

Comment:
```suggestion
        meshFile.mappingName == null ? mappingName : null,
```
(possibly easier to read if the comparison is not negated? not sure if you need a third =)